tutorials/install.md

```diff
@@ -85,6 +124,44 @@ Build and install as follows:
     make -j4
     sudo make install
 
+#### Windows
+
+Open a Visual Studio Command Prompt (search for "x64 Native Tools Command Prompt for VS 2019" in the Windows search near the Start button).
+Right-click and run as Administrator.
+Optionally, right-click and pin to the task bar for quick access in the future.
+
+If you did not add Conda to your ``PATH`` environment variable during Conda installation, you may need to navigate to the location of ``condabin`` in order to use the `conda` command.
```

Comment:
Is there a particular reason you used double quotes?  Just wondering as most other Ignition docs use single quotes.

Reply:
I remember seeing in some places that the single tick didn't render as verbatim, and having to change to double ticks. I don't remember where that happened, so I just used double ticks to avoid having to fix them, ha. I double-checked the tutorial webpages and the single ticks seem to render correctly. I will change to single ticks.